### ISSUE TITLE
Unable to install on WSL Ubuntu

### ISSUE THREAD

**Issue by BMitrovic17** [uhop/node-re2#87]
Install doesn't find needed release for the target OS and goes to rebuild from source thus failing on that as well.

If complete logs are needed i'll be happy to provide them.

```
node -v 15.2.0
npm -v 7.0.8
node-gyp -v 7.1.2
```

```
npm ERR! code 1
npm ERR! path /mnt/x/repository/node-webui/node_modules/re2
npm ERR! command failed
npm ERR! command sh -c install-from-cache --artifact build/Release/re2.node --host-var RE2_DOWNLOAD_MIRROR || npm run rebuild
npm ERR! No github repository was identified.
npm ERR! Building locally ...
```

**Comment by uhop** [comment]
Well, it works as expected failing gracefully.

The reason for failing is that NPM 7 changes how it uses environment variables. We need to support it. As a matter of fact, I started to work on it today.

**Comment by uhop** [comment]
It should be fixed in 1.15.9. Please let me know if it works for you.

**Comment by BMitrovic17** [comment]
Thanks. Works like a charm once again :)
